src/ng/components/selection/selection.core.js

```diff
@@ -0,0 +1,84 @@
+import Directive from '../directive';
+import {VIEW_CORE_NAME, SELECTION_CORE_NAME} from '../../../definition';
+import Event from '../../../core/infrastructure/event';
+//import angular from 'angular';
+
+class SelectionCore extends Directive(SELECTION_CORE_NAME, {view: `^^${VIEW_CORE_NAME}`}) {
+	constructor($scope, $element, $attrs) {
+		super();
+
+		this.$scope = $scope;
+		this.$element = $element;
+		this.$attrs = $attrs;
+		
+		this.selectedItems = new Set();
```

Comment:
it looks like it should be part of selectionModel; selection model already contains that stuff